modular_splurt/code/datums/traits/negative.dm

```diff
@@ -192,3 +192,25 @@
 	deltimer(timer)
 	timer = null
 	timer = addtimer(CALLBACK(src, .proc/crave), craving_after, TIMER_STOPPABLE)
+
+/datum/quirk/nudist
+	// Mostly derived from masked_mook.
+	// Spawning with a gear harness is preferable, but failed during testing.
+	name = "Nudist"
+	desc = "Wearing most types of clothing unnerves you. Bring a gear harness!"
+	gain_text = "<span class='notice'>You feel spiritually connected to your natural form.</span>"
+	lose_text = "<span class='notice'>It feels like clothing could fit you comfortably.</span>"
+	medical_record_text = "Patient expresses a psychological need to remain unclothed."
+	value = -2
```

Comment:
This should probably be a neutral quirk, because since all it does is add a mood change it can easily be used for free quirk powergaming